papers/Vvmax/RLFs.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from scipy.integrate import quad
+
+global E0
+# normalisation energy - no physical meaning, just makes units more sensible
+E0=1e22
+
+
+def make_first_guess(E,L, params,method):
+    """
+    Makes a first guess by setting the amplitude
+    to that of the first bin
+    
+    Params are always gamma,Emax
+    """
+    if method==0:
+        mag = Schechter(E[0],1.,params[0],params[1])
+        guess = [L[0]/mag,params[0],params[1]]
+    elif method==1:
+        mag = integrateSchechter(E[0],1.,params[0],params[1])
+        guess = [L[0]/mag,params[0],params[1]]
+    elif method==2:
+        # returns the log10 magnitude expected with an amplitude of unity
+        mag = logIntegrateSchechter(np.log10(E[0]),1.,params[0],params[1])
+        guess = [L[0]/10.**mag,params[0],params[1]]   
+    else:
+        mag = power_law(E[0],1.,params[0],params[1])
+        guess = [L[0]/mag,params[0],params[1]]
+    
+    
+    return guess
+    
+    
+
+
+def logIntegrateSchechter(log10E,A,gamma,log10Emax):
```

Comment:
I'm not going to nit pick the doc strings unless these wind up 
in the main Repo folder (i.e. `zdm`).  

So if you move them there (it seems you probably should), then 
add short descriptions of each input paraemter

Reply:
I've added proper doc strings to the routines that have been shifted to zdm/vvmax.py . Or, at least, *better* doc strings!